tests/integration/long/test_custom_payload.py

```diff
@@ -0,0 +1,216 @@
+
+    def test_custom_query_basic(self):
+        """
+        Test to validate that custom payloads work with simple queries
+
+        creates a simple query and ensures that custom payloads are passed to C*. A custom
+        query provider is used with C* so we can validate that same custom payloads are sent back
+        with the results
+
+
+        @since 2.2
+        @jira_ticket PYTHON-280
+        @expected_result valid custom payloads should be sent and received
+
+        @test_category queries:basic
+        """
+
+        # Create a simple query statement a
+        query = "SELECT * FROM system.local"
+        statement = SimpleStatement(query)
+        # Validate that various types of custom payloads are sent and received okay
+        self.validate_various_custom_payloads(statement=statement)
+
+    def test_custom_query_batching(self):
+        """
+        Test to validate that custom payloads work with batch queries
+
+        creates a batch query and ensures that custom payloads are passed to C*. A custom
+        query provider is used with C* so we can validate that same custom payloads are sent back
+        with the results
+
+
+        @since 2.2
+        @jira_ticket PYTHON-280
+        @expected_result valid custom payloads should be sent and received
+
+        @test_category queries:batch
+        """
+
+        # Construct Batch Statement
+        batch = BatchStatement(BatchType.LOGGED)
+        for i in range(10):
+            batch.add(SimpleStatement("INSERT INTO test3rf.test (k, v) VALUES (%s, %s)"), (i, i))
+
+        # Validate that various types of custom payloads are sent and received okay
+        self.validate_various_custom_payloads(statement=batch)
+
+
+
+
+    def test_custom_query_prepared(self):
+        """
+        Test to validate that custom payloads work with prepared queries
+
+        creates a batch query and ensures that custom payloads are passed to C*. A custom
+        query provider is used with C* so we can validate that same custom payloads are sent back
+        with the results
+
+
+        @since 2.2
+        @jira_ticket PYTHON-280
+        @expected_result valid custom payloads should be sent and received
+
+        @test_category queries:binding
+        """
+
+        # Construct prepared statement
+        prepared = self.session.prepare(
+            """
+            INSERT INTO test3rf.test (k, v) VALUES  (?, ?)
+            """)
+
+        bound = prepared.bind((1, None))
+
+        # Validate that various custom payloads are validated correctly
+        self.validate_various_custom_payloads(statement=bound)
+
+    def validate_various_custom_payloads(self, statement):
+        """
+        This is a utility method that given a statement will attempt
+        to submit the statement with various custom payloads. It will
+        validate that the custom payloads are sent and received correctly.
+
+        :param statement: to validate the custom queries in conjunction with
```

Comment:
So I got rid of the all the superfluous return statements, is there something else here that needs to change?